--- /etc/mail/spamassassin/v320.pre
+++ /test/etc/spamassassin-3.12/v320.pre
@@ -50,7 +50,7 @@
 
 # Rule2XSBody - speedup by compilation of ruleset to native code
 #
-# loadplugin Mail::SpamAssassin::Plugin::Rule2XSBody
+loadplugin Mail::SpamAssassin::Plugin::Rule2XSBody
 
 # ASN - Look up the Autonomous System Number of the connecting IP
 # and create a header containing ASN data for bayes tokenization.
